package.json

```diff
@@ -74,21 +74,21 @@
         "dotenv": "^16.3.1",
         "eslint": "^8.51.0",
         "eslint-config-prettier": "^9.0.0",
-        "eslint-plugin-svelte": "^2.34.0",
+        "eslint-plugin-svelte": "^2.43.0",
```

Comment:
Including some upgrades to various development-only svelte dependencies. This was to get the generics fixed in one of the files.